src/app/components/form/form.component.ts

```diff
@@ -239,7 +239,7 @@ export class FormComponent implements OnInit, OnChanges, OnDestroy {
     if (domain == '.') {
       return domain;
     } else {
-      return domain.replace(/[ \.]+$/, '');
+      return domain.replace(/\.$/, '');
```

Comment:
The PR description says that trailing spaces should be stripped, but it looks to me that's not being done anymore.

Reply:
It is actually performed before, as it is should also be done for the root.

https://github.com/zonemaster/zonemaster-gui/blob/5a2cd7e4c998ca5981c84bd80797cd640f142b57/src/app/components/form/form.component.ts#L238